csv/src/main/scala/akka/stream/alpakka/csv/scaladsl/CsvFraming.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.csv.scaladsl
+
+import akka.NotUsed
+import akka.event.Logging
+import akka.stream.alpakka.csv.CsvParser
+import akka.stream.scaladsl.Flow
+import akka.stream.stage.{GraphStage, GraphStageLogic, InHandler, OutHandler}
+import akka.stream.{Attributes, FlowShape, Inlet, Outlet}
+import akka.util.ByteString
+
+import scala.util.control.NonFatal
+
+/** Provides CSV framing stages that can separate CSV lines from incoming [[ByteString]] objects. */
+object CsvFraming {
+
+  val backslash: Byte = '\\'
+  val comma: Byte = ','
+  val doubleQuote: Byte = '"'
+
+  def lineScanner(delimiter: Byte = comma,
+                  quoteChar: Byte = doubleQuote,
+                  escapeChar: Byte = backslash): Flow[ByteString, List[ByteString], NotUsed] =
+    Flow[ByteString].via(new GraphStage[FlowShape[ByteString, List[ByteString]]] {
```

Comment:
I'd rather see an actual class than anonymous in the factory like this (and that is the pattern in the other connectors in Alpakka).